sources/install/package_osint.sh

```diff
@@ -401,6 +403,7 @@ function install_trevorspray() {
 
 # Package dedicated to osint, recon and passive tools
 function package_osint() {
+    apt update
```

Comment:
This shouldn't be needed. Please synchronize your branch with `dev`'s latest changes, among other things they introduce an `apt-get update` at the start of every final image dockerfile

Reply:
Synchronizing can be done as follows
```bash
git clone "git@github.com:Neptunium931/Exegol-images"
cd Exegol-images
git remote add upstream "git@github.com:ThePorgs/Exegol-images"
git fetch upstream
git checkout "debian12"
git merge --no-edit "upstream/dev"
# solve conflicts if any
git push
```